File: src/main/java/com/github/Doomsdayrs/Jikan4java/types/Main/User/MangaStats.java
package com.github.Doomsdayrs.Jikan4java.types.Main.User;

import com.fasterxml.jackson.annotation.JsonProperty;

/*
 * Licensed to the Apache Software Foundation (ASF) under one
 * or more contributor license agreements.  See the NOTICE file
 * distributed with this work for additional information
 * regarding copyright ownership.  The ASF licenses this file
 * to you under the Apache License, Version 2.0 (the
 * "License"); you may not use this file except in compliance
 * with the License.  You may obtain a copy of the License at
 *
 *   http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing,
 * software distributed under the License is distributed on an
 * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
 * ====================================================================
 * Jikan4java
 * 03 / November / 2018
 *
 * @author github.com/doomsdayrs
 */
public class MangaStats {
    @JsonProperty("days_read")
    public float days_read;
    @JsonProperty("mean_score")
    public float mean_score;
    @JsonProperty("reading")
    public int reading;
    @JsonProperty("completed")
    public int completed;
    @JsonProperty("on_hold")
    public int on_hold;
    @JsonProperty("dropped")
    public int dropped;
    @JsonProperty("plan_to_read")
    public int plan_to_read;
    @JsonProperty("total_entries")
    public int total_entries;
    @JsonProperty("reread")
    public int reread;
    @JsonProperty("chapters_read")
    public int chapters_read;
    @JsonProperty("volumes_read")
    public int volumes_read;
    

    @Override
    public String toString() {
        return "MangaStats{" +
                "days_read=" + days_read +
                ", mean_score=" + mean_score +
                ", reading=" + reading +
                ", completed=" + completed +
                ", on_hold=" + on_hold +
                ", dropped=" + dropped +
                ", plan_to_read=" + plan_to_read +
                ", total_entries=" + total_entries +
                ", reread=" + reread +
                ", chapters_read=" + chapters_read +
                ", volumes_read=" + volumes_read +
                '}';
    }
}

File: src/main/java/com/github/doomsdayrs/jikan4java/model/main/user/MangaStats.kt
package com.github.doomsdayrs.jikan4java.model.main.user

import com.fasterxml.jackson.annotation.JsonProperty

/** @noinspection ALL
 */
/*
 * This file is part of Jikan4java.
 *
 * Jikan4java is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * Jikan4java is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with Jikan4java.  If not, see <https://www.gnu.org/licenses/>.
 * ====================================================================
 * Jikan4java
 * 03 / November / 2018
 *
 * @author github.com/doomsdayrs
 */
data class MangaStats(
		@field:JsonProperty("days_read") var days_read: Float = 0f,
		@field:JsonProperty("mean_score") var mean_score: Float = 0f,
		@field:JsonProperty("reading") var reading: Int = 0,
		@field:JsonProperty("completed") var completed: Int = 0,
		@field:JsonProperty("on_hold") var on_hold: Int = 0,
		@field:JsonProperty("dropped") var dropped: Int = 0,
		@field:JsonProperty("plan_to_read") var plan_to_read: Int = 0,
		@field:JsonProperty("total_entries") var total_entries: Int = 0,
		@field:JsonProperty("reread") var reread: Int = 0,
		@field:JsonProperty("chapters_read") var chapters_read: Int = 0,
		@field:JsonProperty("volumes_read") var volumes_read: Int = 0
)